canonical-websites/www.ubuntu.com
# Remove rogue links download

## Done
Remove two links from the download menu.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Click on the download menu
- Check the test demo links have been removed

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/3376
